apps/studio/components/interfaces/SQLEditor/UtilityPanel/Results.tsx

```diff
@@ -55,14 +55,18 @@ const Results = ({ id, rows }: { id: string; rows: readonly any[] }) => {
     return <div className="flex h-full items-center justify-center font-mono text-xs">{name}</div>
   }
 
+  const EST_CHAR_WIDTH = 8.25
   const columns: CalculatedColumn<any>[] = Object.keys(rows?.[0] ?? []).map((key, idx) => ({
     idx,
     key,
     name: key,
     resizable: true,
     parent: undefined,
     level: 0,
-    width: 120,
+    width:
+      typeof rows[0][key] === 'string' || typeof rows[0][key] === 'number'
+        ? Math.min(String(rows[0][key]).length * EST_CHAR_WIDTH, 500)
```

Comment:
a better way is probably to consider _all_ rows - take the longest value from all the rows and use it as the multiplier with est_char_width

otherwise e.g if the first row for a column only has 1 char, that column will be extra tiny